tests/MJanssen/Provider/RoutingServiceProviderTest.php

```diff
@@ -30,12 +30,12 @@ public function testApplicationRoutes()
         $app = new Application();
```

Comment:
I think your missing a test case asserting that the routename is actually what you specified